x11-misc/xkeyboard-config/xkeyboard-config-9999.ebuild

```diff
@@ -28,7 +28,11 @@ BDEPEND="
 	dev-libs/libxslt
 	sys-devel/gettext
 	test? (
+		x11-apps/xkbcomp
+		x11-libs/libxkbcommon
 		$(python_gen_any_dep '
+			dev-python/pycountry[${PYTHON_USEDEP}]
```

Comment:
I think you need python_check_deps here.

Reply:
yep